### PR TITLE
feat: onboard microsoft clarity destination

### PR DESCRIPTION
**Pull request by mihir-4116**
## PR Description

Please include a summary of the change along with the relevant motivation and context.

## Notion ticket

Ticket link

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

- - -
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/749)

### CODE REVIEW

File: src/integrations/MicrosoftClarity/browser.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    console.log('inside sdk');
+    (function (c, l, a, r, i, t, y) {
+      c[a] =
+        c[a] ||
+        function () {
+          (c[a].q = c[a].q || []).push(arguments);
+        };
+      t = l.createElement(r);
+      t.async = 1;
+      t.src = 'https://www.clarity.ms/tag/' + i;
```
Comment (MoumitaM) [inline src/integrations/MicrosoftClarity/browser.js:25]: Add `t.setAttribute('data-loader', LOAD_ORIGIN);`

File: src/integrations/MicrosoftClarity/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import MicrosoftClarity from './browser';
+
+export default MicrosoftClarity;
```
Comment (MoumitaM) [inline src/integrations/MicrosoftClarity/index.js:3]: Please change it to a named export

**Review by MoumitaM — CHANGES_REQUESTED** [review]
Also add entry to config_to_integration_names.js file

File: src/integrations/MicrosoftClarity/browser.js (hunk 1 above)
Comment (mihir-4116) [reply src/integrations/MicrosoftClarity/browser.js:26]: updated

File: src/integrations/MicrosoftClarity/index.js (hunk 2 above)
Comment (mihir-4116) [reply src/integrations/MicrosoftClarity/index.js]: updated

**Review by MoumitaM — APPROVED** [review]

**Pull request merged**